src/core/server/config/ensure_valid_configuration.ts

```diff
@@ -9,22 +9,27 @@
 import { ConfigService } from '@kbn/config';
 import { CriticalError } from '../errors';
 
+const ignoredPaths = ['dev.', 'elastic.apm.'];
```

Comment:
The `dev` schema was only added to avoid validation failure. Now that we have this ignore list instead, I removed the `dev` schema and added it here.